sphinx/tutorial/cylc/furthertopics/inheritance.rst

```diff
@@ -0,0 +1,603 @@
+following values:
+
+* Turbine - kerosene
+* Internal Combustion - petrol
+* Human - pizza
+
+Add lines to the ``runtime`` section to represent these four families.
+
+.. spoiler:: Solution warning
+
+   .. code-block:: cylc
+
+          [[ENGINE]]
+          [[TURBINE_ENGINE]]
+              inherit = ENGINE
+              [[[environment]]]
+                  FUEL = kerosene
+          [[INTERNAL_COMBUSTION_ENGINE]]
+              inherit = ENGINE
+              [[[environment]]]
+                  FUEL = petrol
+          [[HUMAN_ENGINE]]
+              inherit = ENGINE
+             [[[environment]]]
+                  FUEL = pizza
+
+We now need to make the three aircraft inherit from one of the three engine
+families. The aircraft use the following types of engine:
+
+* A380 - turbine
+* R44 - internal combustion
+* V22 - turbine
+
+Modify the three tasks so that they inherit from the relevant engine families.
+
+.. spoiler:: Solution warning
+
+   .. code-block:: cylc
+
+         [[a380]]
+             inherit = AIRPLANE, TURBINE_ENGINE
+             [[[meta]]]
+                 title = Airbus A380 Jumbo-Jet
+         [[r44]]
+             inherit = HELICOPTER, INTERNAL_COMBUSTION_ENGINE
+             [[[meta]]]
+                 title = Robson R44 Helicopter
+         [[v22]]
+             inherit = AIRPLANE, HELICOPTER, TURBINE_ENGINE
+             [[[meta]]]
+                 title = V-22 Ofsprey Military Aircraft
+
+Penny Farthing
+^^^^^^^^^^^^^^
+
+Next we want to add a new type of vehicle, an old-fashioned bicycle called a
+penny farthing.
+
+.. image:: https://upload.wikimedia.org/wikipedia/commons/a/a7/Ordinary_bicycle01.jpg
+   :width: 300px
+   :alt: Penny Farthing Bicycle
+   :align: center
+
+To do this we will need to add a new classification of vehicle ``LAND_VEHICLE``,
+and a new subclass of vehicle called ``BICYCLE`` and a new task called
+``penny_farthing`` related in the following manner:
+
+.. digraph:: Example
+   :align: center
+
+   bgcolor=none
+
+   VEHICLE [color=royalblue, fillcolor=powderblue, shape=box, style=filled]
+   LAND_VEHICLE [color=royalblue, fillcolor=powderblue, shape=box, style=filled]
+   BICYCLE [color=royalblue, fillcolor=powderblue, shape=box, style=filled]
+   HUMAN_ENGINE [color=royalblue, fillcolor=powderblue, shape=box, style=filled]
+   penny_farthing [color=royalblue, fillcolor=powderblue, shape=box, style=filled]
+   VEHICLE -> LAND_VEHICLE -> BICYCLE -> penny_farthing
+   HUMAN_ENGINE -> penny_farthing
+
+Add lines to the ``runtime`` section to represent the two new families and one
+task outlined above.
+
+Add a description (``[meta]description``) to the ``LAND_VEHICLE`` and
+``BICYCLE`` families and a title (``[meta]title``) to the ``penny_farthing``
+task.
+
+.. spoiler:: Solution warning
+
+   .. code-block:: cylc
+
+         [[LAND_VEHICLE]]
+             inherit = VEHICLE
+             [[[meta]]]
+                 description = A vehicle which can travel over the ground
```

Comment:
As with line 45, a full-stop at the end of every description sentence would maintain consistency.